pandas/core/groupby/groupby.py

```diff
@@ -1707,7 +1714,7 @@ def ngroup(self, ascending=True):
 
         See Also
         --------
-        .cumcount : Number the rows in each group.
+        cumcount : Number the rows in each group.
```

Comment:
Did you check if this is a function, or a method? If it's a method we should have the class before.